components/animation/index.js

```diff
@@ -77,7 +81,7 @@ AFRAME.registerComponent('animation', {
     }
 
     // Stop previous animation.
-    this.pauseAnimation();
+    // this.pauseAnimation();
```

Comment:
We shouldn't run the config/anime setup in the update block if only an event listener was changed. If only an event listener was changed, we can just can remove/addEventListeners. You can check what changed by AFRAME.utils.diff(this.data, oldData) where oldData is the first argument of update